autoscale/as_meta.go

```diff
@@ -382,6 +392,14 @@ func (c *AutoScaleMeta) Resume(tenant string, tsContainer *TimeSeriesContainer)
 	}
 }
 
+func (c *AutoScaleMeta) GetTenantState(tenant string) (bool, int32, int) {
+	v, ok := c.tenantMap[tenant]
```

Comment:
tenantMap need be protected by mutex

Reply:
But after changing, will the `addPodIntoTenant` and `removePodFromTenant` affect the GetState()? Because they will keep the lock too.